src/en/juju-concepts.md

```diff
@@ -0,0 +1,98 @@
+Title: Juju concepts
+
+
+!!! Note: This page is a work in progress, as part of the work towards 
+documenting Juju 2.0.
+
+# Juju concepts
+
+## The Juju client
+
+The Juju package which you install and run from your local computer is sometimes
+referred to as the Juju client. This is the software which manages your 
+connection to Juju controllers in the cloud, and from which you issue commands
+to deploy and manage services.
+
+## Clouds
+
+In Juju terms, the cloud is the environment which provides resources to be used
+by Juju - machines and storage. This includes traditional public clouds, such
+as Amazon Web Services, Google Compute Engine and Microsoft Azure, as well as
+(possibly private) clouds such as OpenStack.
+
+It also includes things which may not strictly speaking be clouds, but Juju can
+treat as a cloud, such as [MAAS][maas] and your local computer using LXD.
+
+
+## The Juju controller
+
+The controller is a special instance which Juju will create in a cloud when you
+run the bootstrap command. The controller is effectively a Juju agent which runs
+inside the cloud, and it maintains the models that you create there. You can run
+multiple controllers with Juju, and each can maintain multiple models.
+
+## Models
+
+A model is the space within which you can deploy services. A controller can 
+manage several models, and the models themselves can even be shared amongst
+multiple users.
+
+## Charms
+
+The magic behind Juju is a collection of software components called Charms. They
+contain all the instructions necessary for deploying and configuring 
+cloud-based services. The charms publicly available in the online Charm Store 
+represent the distilled DevOps knowledge of experts. Charms make it easy to 
+reliably and repeatedly deploy services, then scale up as required with minimal 
+effort.
+
+## Relations
+
+Few services are designed to be run in isolation. Juju charms also contain the
+knowledge of how to connect services together using common interfaces they
+share. For example, WordPress supports an 'http' interface (for serving the 
+website) and a 'db' interface (for the database which stores the content of
+the site). Any other service which can support these interfaces(e.g haproxy for
+the website and MySQL for the database) can connect to the WordPress charm in 
+a meaningful way.
+
+## Bundles
+
+A bundle is a ready-to-run collection of services that have been modelled to 
```

Comment:
I thought about it, but that's exactly what they are - like a snapshot of a model